forte/data/data_store.py

```diff
@@ -286,6 +311,20 @@ def __setstate__(self, state):
         self._DataStore__tid_idx_dict = {}
         self._DataStore__deletion_count = {}
 
+        # Update `_type_attributes` to store the types of each
+        # entry attribute as well.
+        for tn in self._type_attributes:
+            entry_type = self._add_entry_types(tn)
+            for attr, type_val in entry_type.items():
+                try:
+                    info_dict = self._type_attributes[tn][
+                        constants.TYPE_ATTR_KEY
+                    ][attr]
+                except KeyError:
```

Comment:
Does this mean that `entry_type` has some new attributes that are not present in `self._type_attributes`?

Reply:
Yes

Reply:
Maybe it's better to add some comments here on why skipping them and when this may happen.